### DISCUSSION TITLE
How do you (grey/gray) out a slash command (disable it based on permissions, a role, etc)

### DISCUSSION THREAD

**Discussion by ZoomGC** [Pycord-Development/pycord#1556] [Q&A] [answered]
I was wondering how you can grey out a slash command in the actual command selector on discord, I know you can stop them from running the code in the command easily with `commands.has_permissions()` but it does not stop the actual command itself being ran.

I would also like to know if there is a way to disable it based on a user's ID on Discord and just any commands regarding disabling (greying out) a slash command

**Comment by Lulalaby** [comment, marked as answer]
That doesn't work due to the permission system.

**Reply by ZoomGC** [reply]
Oh I thought that was possible, I have seen it with other bots.. strange

Could you explain this to me if you do not mind

**Reply by Lulalaby** [reply]
https://discord.com/blog/slash-commands-permissions-discord-apps-bots
